src/mcp_jupyter/server.py

```diff
@@ -313,7 +336,7 @@ def query_notebook(
 @NotebookState.refreshes_state
 def _query_view_source(
     notebook_path: str,
-    execution_count: Optional[Union[str, int]] = None,
+    execution_count: Optional[Union[str, int, float]] = None,
```

Comment:
See above comment

Reply:
Any reason not to match the function signature of the tool above where `execution_count` is only an `int`?